finetune/encoding/target_encoders.py

```diff
@@ -192,7 +192,7 @@ def __init__(self, pad_token):
         self.lookup = None
 
     def fit(self, labels):
-        self.classes_ = list(set(lab_i["label"] for lab in labels for lab_i in lab) | {self.pad_token})
+        self.classes_ = sorted(list(set(lab_i["label"] for lab in labels for lab_i in lab) | {self.pad_token}))
```

Comment:
Not sure this necessarily has to be sorted but it threw me off in testing.  Think it might be handy